src/devices/llama_devices.ts

```diff
@@ -0,0 +1,169 @@
+import {Definition, Tz, Fz} from '../lib/types';
+import fz from '../converters/fromZigbee';
+import * as exposes from '../lib/exposes';
+import * as reporting from '../lib/reporting';
+
+const e = exposes.presets;
+const ea = exposes.access;
+const prefixEnpointName: string = 'button_';
+const endpointCount = 4;
+
+type Config = {
+    attribute: string,
+    name: string,
+    description: string,
+    values: ConfigValues[]
+};
+type ConfigValues = {
+    name: string,
+    value: number,
+    command?: string
+};
+const switchTypeConf: Config = {
+    attribute: 'switchType',
+    name: 'switch_type',
+    description: 'Operation method of the switch',
+    values: [
+        {
+            name: 'toggle',
+            value: 0,
+        },
+        // {
+        //     name: 'momentary',
+        //     value: 1
+        // },
+        {
+            name: 'alternate',
+            value: 2,
+        },
+    ],
+};
+const switchAtionConf: Config = {
+    attribute: 'switchActions',
+    name: 'switch_actions',
+    description: 'Command to be generated when the switch moves between its two states',
+    values: [
+        {
+            name: 'on/off',
+            value: 0,
+        },
+        {
+            name: 'off/on',
+            value: 1,
+        },
+        {
+            name: 'toggle',
+            value: 2,
+        },
+    ],
+};
+const configs: Config[] = [
+    switchTypeConf,
+    switchAtionConf,
+];
+
+function addEndpointPrefix(ep: number) {
+    return `${prefixEnpointName}${ep}`;
+}
+
+function createSwitchExposes(epc: number) {
```

Comment:
Instead of doing this, please create a modernExtend for this, [example](https://github.com/Koenkk/zigbee-herdsman-converters/blob/6b3911a45601e5b01e916c74bcf51b9848d2e757/src/lib/lumi.ts#L1305)

Reply:
less code and still works. thanks